### ISSUE TITLE
No longer working?

### ISSUE THREAD

**Issue by Zzegfried** [braddwalker/TeslaInventoryNet#5]
Looks like the API may have changed.

**Comment by braddwalker** [comment]
Yeah, looks like they've implemented bot protection which is blocking the call to the API endpoint. Will poke at it a bit but this might be a dealbreaker.
